osclib/conf.py

```diff
@@ -258,4 +213,4 @@ def apply_remote(self, api):
             self.populate_conf()
         elif config is None:
             # Write empty config to allow for caching.
-            api.dashboard_content_save('config', '')
+            api.attribute_value_save('Config', '')
```

Comment:
The caching rules need to be updated to make this useful. Also this will now insert blank configs in gnome project and similar. I don't think the remote-config toggle should be removed.

Reply:
Actually writing the empty value is pointless anyway - and reading attributes is like 800% faster than reading files. So possibly we just don't cache the config.

I just don't see the value of not having remote config (read)